### ISSUE TITLE
Beanbags 1 shot mechanoids

### ISSUE THREAD

**Issue by Gariba44** [skyarkhangel/CombatRealism#74]
Mechanoids have no resistance to the muscle spasms effect from beanbag rounds, and fall helpless to the ground on the first shot

**Comment by skyarkhangel** [comment]
yep. i am looking to fix this error. But in first, its vanilla bug, that Mechanoids oneshots from all damagedefs which work through hediff.
